package.json

```diff
@@ -9,6 +9,7 @@
   ],
   "scripts": {
     "build": "turbo run build",
+    "dev": "turbo run dev",
```

Comment:
when running yarn dev from root only the commons build runs successfully. The rest type errors:

example:
```
rpch-sdk:dev: src/index.ts(1,65): error TS7016: Could not find a declaration file for module 'rpch-common'.
```